ui/component/page/view.jsx

```diff
@@ -73,11 +73,10 @@ function Page(props: Props) {
   const isLandscapeRotated = useIsMobileLandscape();
   const [sidebarOpen, setSidebarOpen] = usePersistedState('sidebar', false);
 
-  const url = pathname.slice(1).replace(/:/g, '#');
+  const urlPath = `lbry://${(pathname + hash).slice(1).replace(/:/g, '#')}`;
```

Comment:
--- Just for documentation ----
- Is the `hash` (i.e. `pathname + hash`) required for `parseURI`?  
  - Or perhaps it's a misunderstanding from one of my past comments?
  - But looking at the change in `bf158ad6`, the original code does put the `hash` into `uri` and passes it to the router.  So, if the `hash` is important LBRY URI, why it is removed in `app/view.jsx`, but then included here?
    - My shallow understanding is that `hash` is only needed for navigation and sub-sectioning, and not part of LBRY URI, so a bit confused with the mixture and comparison with old code.

Reply:
Ok, confirmed this is an issue.
- oauth branch, logged in via sso
- http://localhost:9090/@Odysee:8
- F5
- It should temporarily have a `#state=12345` while it's redirecting, but should end up back in http://localhost:9090/@Odysee:8.  However, this this change (I think), the hash is put back in.

Reply:
Let me know if this is better now for your branch, the problem was that using uris with `#` instead of `:` would make it part of the `hash` so it would be missing from URI, so it was needed to separate what is used for routing/navigation, and what for the LBRY uri, that's why the parsing and `pathname + hash` (on `app/view.jsx` it just needs the URI, on `show/view.jsx` it replaces the URL with the canonical URI + location data otherwise timestamps, referrals, etc would be lost)